packages/ui/theme/components/ButtonLink.ts

```diff
@@ -0,0 +1,39 @@
+import { defineStyleConfig } from "@chakra-ui/react";
+
+const ButtonLink = defineStyleConfig({
```

Comment:
Change the component name and file name to "Button" as we are overriding Chakra UI's base "Button" component here